packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -37,5 +37,8 @@
   "force_refresh_sobjects": "SFDX: Refresh SObject Definitions",
   "force_data_soql_query_input_text": "SFDX: Execute SOQL Query...",
   "force_data_soql_query_selection_text":
-    "SFDX: Execute SOQL Query with Currently Selected Text"
+    "SFDX: Execute SOQL Query with Currently Selected Text",
+  "gating_title": "Salesforce Feature Previews",
```

Comment:
Please choose a better name than gating_title.

Reply:
feature_previews_title ??    Unclear to me why this matters as it is strictly internal and describes what is happening accurately...   But I can change it...